Description: wait feature
Origin: debian
Author: Alexander Wirt <formorer@debian.org>
Last-Update: 2007-08-30

--- a/firehol.sh
+++ b/firehol.sh
@@ -5703,6 +5703,38 @@
 	test ${FIREHOL_CONF_SHOW} -eq 1 && show_work_realcmd 3
 }
 
+wait_for_interface() {
+	local iface=$1; shift
+	local timeout=60
+
+	if [ -n "$1" ]; then
+		timeout=$1
+	fi
+
+	local start=`date +%s`
+	local found=0
+
+	while [ "`date +%s`" -lt $(($start+$timeout)) -a $found -eq 0 ]
+	do
+		local addr=`ip addr show $iface 2> /dev/null | awk '$1 ~ /^inet$/ {print $2}'`
+		if [ -n "$addr" ]
+		then
+			found=1
+		fi
+		if [ $found -eq 0 ]
+		then
+			sleep 0.5
+		fi
+	done
+
+	if [ $found -eq 1 ]
+	then
+		# the interface is up
+		return 0
+	else
+		return 1
+	fi
+}
 
 
 # ------------------------------------------------------------------------------
@@ -7074,6 +7106,14 @@
 # ------------------------------------------------------------------------------
 # Run the configuration file.
 
+if [ -n "$WAIT_FOR_IFACE" ]
+then
+	for i in "$WAIT_FOR_IFACE"
+	do
+		wait_for_interface $i
+	done
+fi
+
 enable -n trap			# Disable the trap buildin shell command.
 enable -n exit			# Disable the exit buildin shell command.
 source ${FIREHOL_TMP} "$@"	# Run the configuration as a normal script.
